src/component/providers/UIProvider/UIProviderContainer.tsx

```diff
@@ -0,0 +1,16 @@
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import UIProvider from './UIProvider';
+
+export const UIProviderContainer: React.FC = ({ children }) => {
+    const { uiConfig } = useUiConfig();
+
+    if (!uiConfig.flags) {
+        return null;
+    }
+
+    return (
+        <UIProvider darkmode={uiConfig.flags.DARK_MODE || false}>
```

Comment:
Should this be `enableDarkModeSupport` or something like that? Or does it actually toggle dark mode as well?